src/components/tables/cells/TimeStamp.tsx

```diff
@@ -3,34 +3,52 @@ import { formatDistanceToNow, formatRelative } from 'date-fns';
 import { FormattedDate } from 'react-intl';
 
 interface Props {
-    time: string;
+    time: string | Date;
     enableRelative?: boolean;
+    monthOnly?: boolean;
 }
 
-function TimeStamp({ enableRelative, time }: Props) {
+function TimeStamp({ enableRelative, time, monthOnly }: Props) {
     return (
         <TableCell>
             <Tooltip
                 title={
-                    <FormattedDate
-                        day="numeric"
-                        month="long"
-                        year="numeric"
-                        hour="numeric"
-                        minute="numeric"
-                        second="numeric"
-                        timeZoneName="short"
-                        value={time}
-                    />
+                    monthOnly ? (
+                        <FormattedDate
+                            day="numeric"
+                            month="long"
+                            year="numeric"
+                            value={time}
+                        />
+                    ) : (
```

Comment:
This is fine for now - but I would probably change this to allow consumers to pass in the props they want set/overwritten. Or just created a new component.